src/Microsoft.AspNetCore.Server.HttpSys/FeatureContext.cs

```diff
@@ -168,11 +169,15 @@ string IHttpRequestFeature.Protocol
                 if (IsNotInitialized(Fields.Protocol))
                 {
                     var protocol = Request.ProtocolVersion;
-                    if (protocol.Major == 1 && protocol.Minor == 1)
+                    if (protocol == Constants.V2)
+                    {
+                        _httpProtocolVersion = "HTTP/2";
```

Comment:
No minor version for http2?

Reply:
Nope. https://tools.ietf.org/html/rfc7540 Hypertext Transfer Protocol Version 2 (HTTP/2)